packages/template-retail-react-app/app/commerce-api/shopper-baskets.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2022, Salesforce, Inc.
```
And don't forget the other files!